### PR TITLE
Update tf2pulumi

### PR DESCRIPTION
**Pull request by pgavlin**
- Support multi-language examples in the schema target
- Link in tf2pulumi

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,17 +8,27 @@ require (
...
 )
 
-replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v12.4.3+incompatible
+replace (
+	github.com/Azure/go-autorest => github.com/Azure/go-autorest v12.4.3+incompatible
+	github.com/coreos/etcd => github.com/pulumi/etcd v3.3.18+incompatible
+	github.com/hashicorp/terraform-plugin-sdk => github.com/pulumi/terraform-plugin-sdk v0.0.0-20200416232118-ec806f20dbeb
+)
+
+replace github.com/pulumi/tf2pulumi => ../tf2pulumi
```
Comment (EvanBoyle) [inline go.mod:32]: Looks like these two replaces are leftover from development?

File: build/common.mk [hunk 2]
```diff
@@ -147,7 +147,6 @@ all:: build install lint test_all
 
 ensure::
 	$(call STEP_MESSAGE)
-	echo "GO111MODULE=on go mod tidy"; GO111MODULE=on go mod tidy;
```
Comment (EvanBoyle) [inline build/common.mk:150]: Just curious on the reason for removing this?

**Comment by pgavlin** [comment]
Should've posted this as WIP--hence the replaces and the removal of `go mod tidy`, both of which are for local dev.

**Comment by lukehoban** [comment]
From the diffs, looks like we are adding `name: ` in a lot of places we didn't previously - including where it has the same value as the logical name.  Intentional?

E.g. https://github.com/pulumi-bot/pulumi-azuread/compare/76048a1..0343e75#diff-653e69ce47175546644fc36c188fe9aeR26

**Comment by pgavlin** [comment]
> From the diffs, looks like we are adding name: in a lot of places we didn't previously - including where it has the same value as the logical name. Intentional?

Nope. Digging in now.

**Pull request merged**